examples/readers/src/llamaparse_2.ts

```diff
@@ -1,26 +1,31 @@
-import fs from "fs/promises";
-import { LlamaParseReader } from "llamaindex";
+import { LlamaParseReader, VectorStoreIndex } from "llamaindex";
 
 async function main() {
   // Load PDF using LlamaParse. set apiKey here or in environment variable LLAMA_CLOUD_API_KEY
   const reader = new LlamaParseReader({
     resultType: "markdown",
     language: "en",
+    numWorkers: 3, //Load files in batches of 2
     parsingInstruction:
-      "The provided document is a manga comic book. Most pages do NOT have title. It does not contain tables. Try to reconstruct the dialogue happening in a cohesive way. Output any math equation in LATEX markdown (between $$)",
+      "The provided documents are datasheets and Quick-Installation-Guides for Solplanet's Ai-LB series of batteries. They contain tables and graphics. There is also a lot of technical information. The goal is to extract and structure the knowledge in a coherent way",
   });
-  const documents = await reader.loadData("../data/manga.pdf"); // The manga.pdf in the data folder is just a copy of the TOS, due to copyright laws. You have to place your own. I used "The Manga Guide to Calculus" by Hiroyuki Kojima
+  // Can either accept a single file path an array[] of file paths or a directory path
+  const documents = await reader.loadData("../data/LlamaParseData");
```

Comment:
can we just use all the files in the existing data folder?
```suggestion
  const documents = await reader.loadData("../data");
```
we can add new files, but then we need to take care about the license

Reply:
Yeah sure, could also just split the brk-2022.pdf file into 6 parts and put them into a separate folder to showcase parallel processing if that makes things easier.